### PR TITLE
Add tests for QSize + support division

### PR DESCRIPTION
**Pull request by bluebird75**

### CODE REVIEW

**Comment by bluebird75** [comment]
Improvement over previous PR (already merged) on QSize

File: PyQt5-stubs/QtCore.pyi [hunk 1]
```diff
@@ -10770,14 +10770,17 @@ class QSize(sip.simplewrapper):
...
-    def __iadd__(self, value: 'QSize') -> 'QSize': ...
-    def __imul__(self, value: float) -> 'QSize': ...
-    def __isub__(self, value: 'QSize') -> 'QSize': ...
-    def __mul__(self, value: float) -> 'QSize': ...
     def __ne__(self, value: object) -> bool: ...
+    def __add__(self, value: 'QSize') -> 'QSize': ...
+    def __iadd__(self, value: 'QSize') -> 'QSize': ...
     def __sub__(self, value: 'QSize') -> 'QSize': ...
+    def __isub__(self, value: 'QSize') -> 'QSize': ...
+    def __mul__(self, value: 'float') -> 'QSize': ...
```
Comment (altendky) [inline PyQt5-stubs/QtCore.pyi]: Why the `'` for `float`?

**Comment by altendky** [comment]
Also, changelog.

File: PyQt5-stubs/QtCore.pyi (hunk 1 above)
Comment (bluebird75) [reply PyQt5-stubs/QtCore.pyi]: Copy/Paste leftover. I'll fix this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,7 +22,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 * [#162](https://github.com/python-qt-tools/PyQt5-stubs/pull/162) fixes all method not declared as signals
 * [#184](https://github.com/python-qt-tools/PyQt5-stubs/pull/184) Fix missing module variable
    detected by latest mypy 0.930
-
+* [#183](https://github.com/python-qt-tools/PyQt5-stubs/pull/183) Add missing operations on QSize
+
```
Comment (altendky) [inline CHANGELOG.md]:
```suggestion

```

**Review by altendky — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (bluebird75) [reply CHANGELOG.md]: Oups, probably a side effect of web merging.

**Pull request merged**